server/app.js

```diff
@@ -3,6 +3,9 @@ import express from 'express';
 import cookieParser from 'cookie-parser';
 import logger from 'morgan';
 import indexRouter from './routes/index';
+import createDatabaseConnection from './database/connection';
+
+createDatabaseConnection();
```

Comment:
@sumeetrohra  database connection code is already written in /bin/www file, please remove this line